changelog_entry.yaml

```
@@ -0,0 +1,5 @@
+- bump: {patch}
+  changes:
+    {changed}:
+      - TestGetTracer
+      - tracer_fixture_service
```

__issue, blocking:__
```suggestion
- bump: patch
  changes:
    changed:
    - TestGetTracer
    - tracer_fixture_service
```